analytics/src/main/java/com/segment/analytics/SegmentIntegration.java

```diff
@@ -351,15 +368,99 @@ void performFlush() {
       // Don't use the result of QueueFiles#forEach, since we may not upload the last element.
       payloadsUploaded = payloadWriter.payloadCount;
 
+
+      BufferedReader in;
+      JsonReader jreader;
+
+      try {
+        in = new BufferedReader(new InputStreamReader(connection.connection.getInputStream()));
+        jreader = new JsonReader(in);
+
+        HashMap<String, HashMap<String, String>> responses = new HashMap<String, HashMap<String, String>>();
+
+        jreader.beginObject();
+        jreader.nextName();
+        jreader.beginObject();
+
+        while (jreader.hasNext()){
+          HashMap<String, String> response = new HashMap<String, String>();
+
+          String name = jreader.nextName();
+//          Log.i("SampleApp", "Response name id: " + name);
+          jreader.beginObject();
+          while (jreader.hasNext()){
+            String debugName = jreader.nextName();
+            String debugString;
```

Comment:
If this is debug code, it should be removed or disabled.  If not, the variable names should be changed.